src/reducers/index.js

```diff
@@ -0,0 +1,11 @@
+const createBook = (book) => ({
```

Comment:
- [ ] [REQUIRED] it should combine app specific reducers. 
[screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/ReducxReducers/CombinesAppReducers.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/)